pages/dmp_supporting_info/support_09_responsibilities_resources.md

```diff
@@ -23,17 +23,46 @@ turing:
 
 {% include callout.html type="tip" content="
 **Why is this topic important**\\
->> What is in for you?\\
->> Short & concise\\
->> Add lines as needed\\
+>> Defining responsibilities, project involvement and cost related to research data management is highly important when planning and conducting research projects\\
+>> Relevant estimates of costs for research data management should be included when applying for financing, and this chapter will help you to develop relevant estimates\\
 " %}
 
 ## About this chapter
 
-General information
+In this chapter, you are asked to define responsibilities, who will be involded in the project, and the cost related to research data management. Remeber that additional expertice can be needed when covering the areas of responsibility in the project. Also remeber that cost can be relevant both for facilities, equipment, hardware, software, license costs or other recurring fees, preservation and archiving, costs for preparing data for publication and more. The questions in this chapter cover such aspects of research data management.       
 
 ## Question-specific guidance
 
-### Question 1
+### Have data management responsibilities in the project been assigned?
+Please specify. 
 
-### Question 2
+### To execute the project, is additional expertise required?
+In the context of research data management this could include:
+
+Data Stewards
+Data Engineers
+Software developers
+Data entry specialists
+Data protection specialists
+Legal advisers
+
+### To execute the project, are arrangements needed to access facilities or equipment?
+Necessary arrangements to access facilities or equipment may include training project members, arranging facility access, or executing security clearance. Consider that these processes can be time-consuming, and therfore should be planned for. 
+
+### Will you require hardware or software in addition to what is usually available in the institute?
+Please specify.
+
+### Will there be (software) license costs or other recurring fees to keep data or documents available?
```

Comment:
```suggestion
### Will there be software license costs or other recurring fees to work with data or documents?
```

Reply:
Comment to the KM to avoid 'availability' misunderstandings. Still looking for a better phrasing.